test/integration/containerized/test_container_management.py

```diff
@@ -74,6 +74,10 @@ def test_cancel_will_remove_container(test_data_dir, container_runtime_installed
         cli, container_runtime_installed, 'ansible_runner_foo_bar'
     ), 'Found a running container, they should have all been stopped'
 
+    # clean up settings artifacts generated by test
+    if os.path.exists(env_dir):
+        shutil.rmtree(env_dir)
```

Comment:
If this is necessary, then the test is doing something wrong. At least add a `FIXME` note here that this can be removed once #847 is merged.